main.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"mime/multipart"
+	"net/http"
+	"time"
+	"github.com/warrenbailey/ftp"
+	"github.com/gorilla/mux"
+)
+
+type HealthCheck struct {
+	Status string
+	Ftp string
```

Comment:
Hyper pedantic, but best practice is to keep a consistent case for initials. So FTP or ftp, rather than Ftp. https://github.com/golang/go/wiki/CodeReviewComments#initialisms